webui/react/src/routes/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
 import Admin from 'pages/Admin';
+import Clusters from 'pages/Cluster';
```

Comment:
nit: `import Cluster ...`

Reply:
thanks for catching all these. I did a `git grep -i clusters` and I think these are covered now, other than routes stuff which needs it